### PR TITLE
Added Chess Board

### PR DESCRIPTION
**Pull request by DhanushNehru**

### CODE REVIEW

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Please follow the guidelines in the README.md file - please revert your changes to `src/art.js`.

**Comment by DhanushNehru** [comment]
Changes reverted

File: src/art/DhanushNehru/index.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<html>
+<head>
+</head>
+<body onload="drawChessboard()">
+  <center><canvas id="chessboard" width="700px" height="400px"></canvas></center>
```
Comment (auxfuse) [inline src/art/DhanushNehru/index.html:5]: `center` HTML element is deprecated. Can this be changed please?

**Review by auxfuse — CHANGES_REQUESTED** [review]
Deprecated HTML usage `center` to be updated and can easily be maintained via CSS.

File: src/art/DhanushNehru/index.html (hunk 1 above)
Comment (DhanushNehru) [reply src/art/DhanushNehru/index.html]: Yeah, I removed it

**Comment by DhanushNehru** [comment]
Hi @sophiabrandt / @auxfuse can someone review this ?

**Pull request merged**